tests/Application/config/packages/test/framework.yaml

```diff
@@ -2,3 +2,10 @@ framework:
     test: ~
     session:
         storage_factory_id: session.storage.factory.mock_file
+
+    mailer:
+        dsn: 'null://null'
```

Comment:
This should be defined as an environment variable.

Reply:
fixed